docs/synapse/userguides/storm_ref_cmd.rstorm

```diff
@@ -200,16 +200,17 @@ The ``count`` command enumerates the number of nodes returned from a given Storm
 .. storm-pre:: [inet:ipv4=66.129.222.1 inet:ipv4=184.82.164.104 inet:ipv4=209.161.249.125 inet:ipv4=69.90.65.240 inet:ipv4=70.62.232.98 +#aka.feye.thr.apt1]
```

Comment:
For command description (line 187) suggest:
The ``count`` command enumerates the number of nodes returned from a given Storm query and displays the final tally. The associated nodes can also be displayed with the ``--yield`` option.

Reply:
updated